src/router.ts

```diff
@@ -310,7 +310,7 @@ export class Router {
    * @param params The route parameters to be used when populating the route pattern.
    * @param options The navigation options.
    */
-  navigateToRoute(route: string, params?: any, options?: NavigationOptions): boolean {
+  navigateToRoute(route: string, params?: any, options?: NavigationOptions): NavigationResult {
```

Comment:
Strangely in my local version, this is what I already had. I wonder when and why they got changed.